src/common/balances.lua

```diff
@@ -1,54 +1,89 @@
+--- Module for managing balances and transactions.
+-- @module balances
+require(".common.types")
```

Comment:
I don't think this is necessary for linting to apply?

Reply:
Apparently not - I asked gpt for a pattern where I could have a types file and it claimed this was necessary - seems untrue.

Reply:
[cc20d60](https://github.com/ar-io/ar-io-ant-process/pull/33/commits/cc20d607014d4895de5c7141d2d32da0052f016f)